src/components/UserProfile/EditableModal/RoleInfoModal.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector, connect } from 'react-redux';
+import { Button, Modal, ModalBody, ModalFooter, ModalHeader, Col, Row} from 'reactstrap';
+import { updateInfoCollection } from '../../../actions/information'
+import { Editor } from '@tinymce/tinymce-react';
+import { boxStyle, boxStyleDark } from 'styles';
+import { useEffect } from 'react';
+import { toast } from 'react-toastify';
+
+const RichTextEditor = ({ disabled, value, onEditorChange }) => (
```

Comment:
Hi Nathalia,

Everything looks good. Could we extract RichTextEditor to its own component so that this file looks cleaner and modular?

Thank you!